main.go

```diff
@@ -137,3 +143,40 @@ func run(kubeConfig string) error {
 
 	return nil
 }
+
+func enableKubeVirtFeatureGateHostDevices() {
+	// set up kubevirtClient for PCIDeviceClaims controller
+	clientConfig := kubecli.DefaultClientConfig(&pflag.FlagSet{})
+	var virtClient kubecli.KubevirtClient
+	virtClient, err := kubecli.GetKubevirtClientFromClientConfig(clientConfig)
```

Comment:
should return when err != nil, 

the next access to `virtClient.KubeVirt(ns).Get(cr, &v1.GetOptions{})` has pre-condition that `virtClient` is valid.